solutions/usaco-894.mdx

```diff
@@ -117,6 +117,28 @@ public class GrassPlanting {
 
 </JavaSection>
 
+<PySection>
+
+```py
+fin = open('planting.in', 'r')
+inputs = [i for i in fin.read().splitlines()]
```

Comment:
wait can't this just be replaced with
```
fin.read().splitlines()
```
or
```
fin.readlines()
```

Reply:
^